include/lsst/jointcal/CcdImage.h

```diff
@@ -72,6 +72,13 @@ class CcdImage {
         getWholeCatalog().copyTo(getCatalogForFit());
     }
 
+    /**
+     * Count the number of valid measured and reference stars that fall within this ccdImage.
+     *
+     * @return (measured, reference) stars in the image.
```

Comment:
Consider adding "number of" before (measured", just to emphasize that this is a pair of counts, not a pair of collections of star objects. It duplicates the brief description, so I certainly won't insist.